backend/app/models/visualization.py

```diff
@@ -24,21 +24,21 @@ def validate_to_json(cls, value):
         return value
 
 
-class VisualizationIn(VisualizationBase):
+class VisualizationDataIn(VisualizationDataBase):
     pass
 
 
-class VisualizationDB(Document, VisualizationBase):
+class VisualizationDataDB(Document, VisualizationDataBase):
     creator: UserOut
     created: datetime = Field(default_factory=datetime.utcnow)
     modified: datetime = Field(default_factory=datetime.utcnow)
     bytes: int = 0
     content_type: ContentType = ContentType()
 
     class Settings:
-        name = "visualizations"
+        name = "visualizationData"
```

Comment:
Do you we use camel case anywhere? Should we use all lower case with underscore between each word instead?

Reply:
I think that will be better. Changed.